Source/Timeline/AllDayView.swift

```diff
@@ -0,0 +1,186 @@
+
+import UIKit
+
+public class AllDayView: UIView {
+  
+  internal weak var eventViewDelegate: EventViewDelegate?
```

Comment:
It's `internal` by default, you are free to omit the modifier

Reply:
I wanted to be explicit to express I didn't want to make this public/open. 

Is a comment more appropriate or just removing `internal` should be good?